tests/unit/ssl.c

```diff
@@ -0,0 +1,112 @@
+/*  vim:expandtab:shiftwidth=2:tabstop=2:smarttab:
+ *
+ *  Drizzle Client & Protocol Library
+ *
+ * Copyright (C) 2018 Sociomantic Tsunami
```

Comment:
This copyright message should just be the normal sociomantic copyright message.

Reply:
Like this? 

>   Copyright: Copyright (c) 2018 sociomantic labs GmbH?